src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -629,15 +637,14 @@ <h2 class="m1">Apple</h2>
     selected
     option="a">ABOUT</div>
   <div role="tabpanel"
-      class="tabContent p1 p">Fruit are righ of vitamins.</div>
+      class="tabContent p1 p">Fruit are rich of vitamins.</div>
```

Comment:
Grammar edit + lorem ipsum:

"Fruit is rich in vitamins and minerals. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."